src/stores/errors/errorsReducer.ts

```diff
@@ -0,0 +1,37 @@
+import * as Redux from 'redux';
+
+import { ErrorsActions } from './errorsActions';
+
+export interface ErrorsStoreState {
+  error: any;
```

Comment:
how would you distinguish one error from another?
I would say you need to have a key-value pair for each API call that might throw, e.g 
getRecipe: unknown,
getRecipes: unknown

etc